coffeelint.json

```diff
@@ -0,0 +1,19 @@
+{
+    "check_scope": {
+      "module": "coffeescope2",
+      "level": "warn",
+      "environments": [],
+      "globals": {},
+      "overwrite": false,
+      "same_scope": false,
+      "shadow": false,
+      "shadow_builtins": false,
+      "shadow_exceptions": [],
+      "undefined": false,
+      "hoist_local": true,
+      "hoist_parent": true,
+      "unused_variables": true,
+      "unused_arguments": false,
+      "unused_classes": false
```

Comment:
I decided to fix this a different way, because it meant having to manually keep this section in sync with resin-lint, not something I want to have to deal with